### DISCUSSION TITLE
Can someone please upload the gerber files so I can download them please?

### DISCUSSION THREAD

**Discussion by Picucu** [peng-zhihui/HelloWord-Keyboard#124] [Q&A]
Hi, the website that was provided will not let me download the Gerber files(loads infinitely), it would be great if someone could upload all the files from the oshwhub.com repository for me. Thank you so much!

**Comment by xuexuedan** [comment]
You can leave an email and I can send it to you.

**Reply by Picucu** [reply]
nooneimportant121@gmail.com
Thank you so much!
